common/remote/sockfile.cpp

```diff
@@ -2885,25 +2967,32 @@ class CAsyncCommandManager
 
 
 
-inline void appendErr(MemoryBuffer &reply, RemoteFileCommandType e)
+inline void appendErr(MemoryBuffer &reply, unsigned e)
 {
-    reply.append((unsigned)e).append(getRFCText(e));
+    reply.append(e).append(getRFSERRText(e));
 }
-inline void appendErr2(MemoryBuffer &reply, RemoteFileCommandType e, unsigned v)
+inline void appendErr2(MemoryBuffer &reply, unsigned e, unsigned v)
 {
     StringBuffer msg;
-    msg.append(getRFCText(e)).append(':').append(v);
-    reply.append((unsigned)e).append(msg.str());
+    msg.append(getRFSERRText(e)).append(':').append(v);
+    reply.append(e).append(msg.str());
+}
+inline void appendErr3(MemoryBuffer &reply, unsigned e, int code, const char *errMsg)
+{
+    StringBuffer msg;
+    msg.appendf("ERROR: %s(%d) '%s'", getRFSERRText(e), code, errMsg?errMsg:"");
+    reply.append(e);
+    reply.append(msg.str());
 }
-inline void appendErr3(MemoryBuffer &reply, RemoteFileCommandType e, int code, const char *errMsg)
+inline void appendCmdErr(MemoryBuffer &reply, unsigned e, int code, const char *errMsg)
```

Comment:
The unsigned is now always a cmd, so would be better as RemoteFileCommandType cmd here.